packages/extension/src/companion/Companion.tsx

```diff
@@ -56,7 +56,7 @@ const Container = ({
   return (
     <div
       ref={containerRef}
-      data-testId="companion"
+      data-testid="companion"
```

Comment:
with this change we just need to ensure it get's removed on build correctly.

Reply:
Actually `data-testId` with uppercase `I` was wrong, since our regex is `^data-testid$`.

Reply:
We do have `data-testid` without uppercase on most other places.

Reply:
Ah ok, just checking, in that case fine 🙏

Reply:
Validated now that it gets removed. Tnx for sanity check 🤘